vault/seal/awskms/awskms.go

```diff
@@ -91,33 +90,7 @@ func (k *AWSKMSSeal) SetConfig(config map[string]string) (map[string]string, err
 		return nil, fmt.Errorf("'kms_key_id' not found for AWS KMS seal configuration")
 	}
 
-	// Check and set region
```

Comment:
The comment for this method around the order of precedence should be updated since it's no longer correct.

Reply:
That's a good idea! I added this comment:
>Please see GetOrDefaultRegion for an explanation of the order in which region is parsed.
The comment on that method itself is quite exhaustive and if I repeat it here, the version here could drift from the one on the method in case of future revisions. 

Let me know if that seems like the best way to approach it.